authentication/views.py

```diff
@@ -1,15 +1,13 @@
 """Authentication views"""
 
 import logging
-from urllib.parse import urljoin
 
+from django.conf import settings
 from django.contrib.auth import logout
 from django.shortcuts import redirect
 from django.utils.http import url_has_allowed_host_and_scheme, urlencode
-from django.utils.text import slugify
 from django.views import View
 
-from main import settings
```

Comment:
Importing from `main` instead of `django.conf` breaks the pytest django fixtures.